deepspeed/module_inject/utils.py

```diff
@@ -0,0 +1,49 @@
+# helper function to map between DS policies and DS containers
+def policy_to_ds_container(**kwargs):
+    container = None
+    policy = kwargs['policy']
+
+    if policy is None:
+        print("Policy is None")
+    else:
+        from .containers import HFGPT2LayerPolicy
+        from .containers import HFBertLayerPolicy
+        from .containers import BLOOMLayerPolicy
+        from .containers import HFGPTJLayerPolicy
+        from .containers import HFGPTNEOLayerPolicy
+        from .containers import GPTNEOXLayerPolicy
+        from .containers import HFOPTLayerPolicy
+        from .containers import MegatronLayerPolicy
+        from .containers import HFDistilBertLayerPolicy
+
+        if isinstance(policy, HFGPT2LayerPolicy):
+            from .containers import DS_GPT2Container
+            container = DS_GPT2Container(**kwargs)
+        elif isinstance(policy, HFBertLayerPolicy):
+            from .containers import DS_BERTContainer
+            container = DS_BERTContainer(**kwargs)
+        elif isinstance(policy, BLOOMLayerPolicy):
+            from .containers import DS_BloomContainer
+            container = DS_BloomContainer(**kwargs)
+        elif isinstance(policy, HFGPTJLayerPolicy):
+            from .containers import DS_GPTJContainer
+            container = DS_GPTJContainer(**kwargs)
+        elif isinstance(policy, HFGPTNEOLayerPolicy):
+            from .containers import DS_GPTNEOContainer
+            container = DS_GPTNEOContainer(**kwargs)
+        elif isinstance(policy, GPTNEOXLayerPolicy):
+            from .containers import DS_GPTNEOXContainer
+            container = DS_GPTNEOXContainer(**kwargs)
+        elif isinstance(policy, HFOPTLayerPolicy):
+            from .containers import DS_OPTContainer
+            container = DS_OPTContainer(**kwargs)
+        elif isinstance(policy, MegatronLayerPolicy):
+            from .containers import DS_MegatronGPTContainer
+            container = DS_MegatronGPTContainer(**kwargs)
+        elif isinstance(policy, HFDistilBertLayerPolicy):
+            from .containers import DS_DistilBERTContainer
+            container = DS_DistilBERTContainer(**kwargs)
```

Comment:
This could made more concise if we move the imports above to where the `*LayerPolicy` are imported and use a dictionary:
```python
    policy_to_container = {
        HFGPT2LayerPolicy: DS_GPT2Container,
        HFBertLayerPolicy: DS_BERTContainer,
        BLOOMLayerPolicy: DS_BloomContainer,
    }
    policy_type = type(kwargs['policy'])
    if policy_type not in policy_to_container:
        logger.error(f"Policy type {policy_type} not supported")
    else:
        container = policy_to_container[policy_type]
        return container(**kwargs)
```

The one caveat with this change is that `isinstance` will return `True` if the provided object is a subclass of the type we are comparing against and this method will not do that, but I don't think that matters here.